openfisca_core/columns.py

```diff
@@ -32,6 +37,7 @@ class Column(object):
     formula_class = None
     is_period_size_independent = False  # When True, value of column doesn't depend from size of period (example: age)
     is_permanent = False  # When True, value of column doesn't depend from time (example: ID, birth)
+    period_behavior = None
```

Comment:
I'm also more leaning towards `period_unit`, which describes the concept better.

We can keep using `period_behavior` for development while we are not sure about the name and may need to do a global research and replace. But before merging, we'll have to come back to a better name.